setup.cfg

```diff
@@ -36,6 +36,7 @@ setup_requires =
 install_requires =
 	gwdatafind
 	ligo-segments
+        scitokens >=1.4.0
```

Comment:
Something looks up with the indentation here, can you double-check that this is a single tab indent, and not spaces?